plugins/by-name/blink-cmp-copilot/default.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "blink-cmp-copilot";
+  package = "blink-cmp-copilot";
+
+  maintainers = [ lib.maintainers.HeitorAugustoLN ];
+
+  callSetup = false;
+  hasLuaConfig = false;
+  hasSettings = false;
+
```

Comment:
Could you add `description = '...'` that says that this plugin is configured through blink.cmp's `sources.providers` ?

Reply:
Done